### PR TITLE
Set a default value for mia_user_path in configuration window

### PR DESCRIPTION
**Pull request by sapetnioc**
When starting populse_mia for the first time without any configuration file, an window is opened asking for a path. If I understand what is expected in that path, we can provide a default path value using module file parent. This allow for new user to simply choose Ok button to start.

### CODE REVIEW

**Comment by servoz** [comment]
This is a thing (one more !) that i would like to work one (the install part of mia), an this is why we keep currently #modules in the root of the mia project. I have some ideas on this part. Is it possible for you for now to only use the python3 main.py instead of python3 -m populse_mia ?

**Comment by sapetnioc** [comment]
Yes. I did not read the doc to know how to properly start. I juste tried
python3 -m and since it worked I thought it was the right way to go.

**Comment by sapetnioc** [comment]
There is no behavior difference when using `main.py`. Personally, I would prefer to use `python3 -m populse_mia` because the user can ignore where mia is installed. On the other hand, it is necessary to know where is located main.py (`python3 /casa/src/populse_mia/python/populse_mia/main.py` in my case). Using `-m` makes it easier to document the mia startup for everybody.

**Comment by servoz** [comment]
In fact, there is a difference when using one way or another.

We built the main module to be able to work with the whole package that hosts it, without any interaction with the version that may be different in the site-packages of the station (it's our little vanity !).

The idea: only developers should use the main module (I've put in my .bashrc : `alias mia='pwd_orig=$PWD; cd /home/econdami/Data/Git_Projects/populse_mia/python/populse_mia; python3 main.py; cd $pwd_orig'`), which must therefore be in the development version (linked to our github). The user version is located in the station's website-packages and must be launched with python3 -m populse_mia.

In short:
- the user, must use the installer, a link exists in the documentation for it (this is the part I would like to make evolve), then types python3 -m populse_mia and that's it.
- The developer, gets a clone of the github project and so knows where the corresponding main module is and launches in developer mode by doing mia (if he has the alias defined in his environment or goes into the directory to launch  python3 main.py.) Aaccording to our discussions (nothing is fixed) this can evolve but for now this is the philosophy adopted.

**Comment by servoz** [comment]
I'm not sure that the change in this PR is currently necessary if we launch mia in developer mode or in user mode. Can I close this PR?

**Comment by sapetnioc** [comment]
I think this discussion goes far beyond this PR and will be in a future meeting. But, to make it short here, there is a third environment for launching mia : [casa_distro](https://github.com/brainvisa/casa-distro). This is our development environments (yes, plural ; there are many Linux, Windows and Mac) that has its own way to install projects from sources. This PR makes things a little bit easier when starting mia from a brain new development environment (even home directory is part of this environment). So if it does not brake the two others I would prefer to keep it. Otherwise, the two options are to close it or I can modify it to be applied only in casa_distro (there are environment variables specific to this environment). Up to you to choose.

**Comment by servoz** [comment]
Yes, it is a topic to be discussed as soon as possible during meetings.

But in fact by using the `__file__` (this PR) as the default directory (or rather two floors above !) for the configuration file, we gives the possibility to want to write personal data in the site-packages and I think it's really not desirable... 

Currently, the automatic installer creates everything user need in the ~/.populse_mia directory.  As I wrote earlier this part of the installation is intended to evolve, but writing personal data in the site -package of the station really doesn't seem desirable to me.

If it is important for you to run mia in user mode (if the code passes in the code part of this PR, it has been designed to work in user mode) to use it for development in the casa_distro environment (which I admit I don't know), please make a change that only works in casa_distro mode and not for normal use. Besides I'm not sure that in this case it will be better because you need somewhere directories of processes, properties and resources directories  (the purpose of the /configuration.yml file is only to indicate where to find these directories), but I don't know casa_distro...

**Comment by sapetnioc** [comment]
Sorry, I understand just now that I did not understand the purpose of that directory at the beginning. If it is to write personal data this PR must be closed. I will keep the corresponding branch until I decide if I propose a new PR dedicated to casa_distro setup.
